pkg/maintainer/spv/spv.go

```diff
@@ -49,18 +57,384 @@ func (sm *spvMaintainer) startControlLoop(ctx context.Context) {
 		}
 
 		select {
-		case <-time.After(sm.config.RestartBackOffTime):
+		case <-time.After(sm.config.RestartBackoffTime):
 		case <-ctx.Done():
 			return
 		}
 	}
 }
 
 func (sm *spvMaintainer) maintainSpv(ctx context.Context) error {
-	logger.Infof("Maintaining SPV proof...")
+	for {
+		if err := sm.proveDepositSweepTransactions(); err != nil {
+			return fmt.Errorf(
+				"error while proving deposit sweep transactions: [%v]",
+				err,
+			)
+		}
+
+		// TODO: Add proving of other type of SPV transactions: redemption
+		// transactions, moving funds transaction, etc.
+
+		select {
+		case <-time.After(sm.config.IdleBackoffTime):
+		case <-ctx.Done():
+			return ctx.Err()
+		}
+	}
+}
+
+func (sm *spvMaintainer) proveDepositSweepTransactions() error {
+	depositSweepTransactions, err := sm.getUnprovenDepositSweepTransactions()
+	logger.Infof(
+		"Found [%d] unproven deposit sweep transaction(s)",
```

Comment:
We should start log messages from a lower-case:

```suggestion
		"found [%d] unproven deposit sweep transaction(s)",
```

Let's align all other log messages in this file.